frontend/src/app/shared/form/oib-scan-mode/oib-scan-mode.component.spec.ts

```diff
@@ -51,7 +54,7 @@ describe('OibScanModeComponent', () => {
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [OibScanModeComponent]
+      imports: [OibScanModeComponent, MockI18nModule, ReactiveFormsModule]
```

Comment:
Is ReactiveFormsModule needed? It should work without it as the component should import it no?